### DISCUSSION TITLE
WLED support to locate drawer in storage organizer

### DISCUSSION THREAD

**Discussion by ghost** [sysadminsmedia/homebox#177] [Ideas]
Hi All,
I have an idea which may suit more people not just me. :) I would like to add a button to Homebox to locate exact drawer which contains item I'm looking for. I googled for a bit and found plugin for inventree [https://github.com/invenhost/inventree-wled-locator](url) and another project [(https://github.com/FireMarshmellow/M.I.M.O.S.A)](url) but I would love to see it in Homebox since I already have approximately 200 small drawers in storage organizers tracked in Homebox. Unfortunately I'm not a coder so I don't know how to achieve something like this.
![organiser](https://github.com/user-attachments/assets/ba419076-137d-4c3e-8e0a-8307f3d0cadc)

**Comment by supaeasy** [comment]
Wow this is the definition of overengineering. I love it!
